### PR TITLE
wpa_supplicant: new article

### PR DESCRIPTION
**Pull request by mcpcpc**

### CODE REVIEW

File: post-installation/wpa_supplicant.txt [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+________________________________________________________________________________
+
+Begin by first verifying that you have wpa_supplicant installed:
+
+$ kiss b wpa_supplicant && kiss i wpa_supplicant
+
+At this point, you will want to create a wpa_supplicant file to store your 
+wireless network information and credentials:
+
+$ mkdir /etc/wpa_supplicant
```
Comment (dylanaraps) [inline post-installation/wpa_supplicant.txt:21]: `mkdir -p`.

**Comment by dylanaraps** [comment]
This is really nice. Thanks

**Comment by dylanaraps** [comment]
I think this makes much more sense to live in a category called `wifi`. I also want to write an `eiwd` page alongside this one. :+1:

**Comment by mcpcpc** [comment]
> I think this makes much more sense to live in a category called `wifi`. I also want to write an `eiwd` page alongside this one. 👍

make sense. moved post-installation->wifi and updated the index.txt file.

File: post-installation/wpa_supplicant.conf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Allow users in the 'wheel' group to control wpa_supplicant  
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=wheel
+
+# Make this file writable for wpa_gui / wpa_cli
+update_config=1
+
+network={
+       ssid="<BSSID>"
+       psk="<PASSWORD>"
+}
```
Comment (dylanaraps) [inline post-installation/wpa_supplicant.conf:10]: This is probably better embedded in the page itself. Keeps things nice and self-contained.

File: post-installation/wpa_supplicant.txt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+The following @/wpa_supplicant.conf can be used as a "starter" configuration
+file. Remember to replace the <BSSID> and <PASSWORD> with your actual wireless
+network credentials. For additional network requirements, refer to the 
+"wpa_supplicant" Arch Linux Wiki page [0].
+
+
+Generating a Passphrase
+________________________________________________________________________________
+
+NOTE: This section is not required but HIGHLY recommend since storing your
```
Comment (dylanaraps) [inline post-installation/wpa_supplicant.txt:33]: `recommend` -> `recommended`

File: post-installation/wpa_supplicant.txt [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      password in clear text is not a good practice.
```
Comment (dylanaraps) [inline post-installation/wpa_supplicant.txt:34]: `not a good practice` -> `not good practice`

File: post-installation/wpa_supplicant.txt [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+________________________________________________________________________________
+
+NOTE: This section is not required but HIGHLY recommend since storing your
+      password in clear text is not a good practice. 
+
+To allow for quicker connections to a network whose BSSID is already known, we
+can make use of wpa_passphrase, a command line tool which generates the minimal
+configuration needed by wpa_supplicant: 
+
+$ wpa_passphrase <BSSID> <PASSWORD>
```
Comment (dylanaraps) [inline post-installation/wpa_supplicant.txt:40]: `<BSSID>` (and all occurrences of `<word>`) will be interpreted as HTML by the browser and won't be displayed. You have two options here, using `&lt;`/`&gt;` or by swapping to a different character.  :+1:

File: post-installation/wpa_supplicant.txt [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+The wireless status connection can be verified by reviewing the output of the
+following command:
+
+$ ifconfig
+
+
+Managed Wireless Connection via DHCPCD
+________________________________________________________________________________
+
+Assuming that dhcpd is already installed, busybox's runsv can be used to create
```
Comment (dylanaraps) [inline post-installation/wpa_supplicant.txt:70]: `dhcpd` -> `dhcpcd`

**Comment by dylanaraps** [comment]
Thanks. :+1:

**Pull request merged**